core/templates/pages/topic-editor-page/topic-editor-page.component.html

```diff
@@ -8,12 +8,14 @@
           [attr.dropdown]="getTotalWarningsCount()"
           class="nav-item icon nav-list-item"
           ngbDropdown
+          role="menu"
           (click)="selectMainTab()">
         <a ngbTooltip="Main Editor"
            placement="bottom"
            class="nav-link navbar-tab"
            tabindex="0"
            aria-label="Main Editor"
+           role="menuitem"
```

Comment:
Should this be `menu-item`?

Reply:
@Rijuta-s No, it is supposed to be "menuitem" see here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role

Reply:
Got it. Thanks